includes/config.php

```diff
@@ -1,128 +0,0 @@
-<?php
-require 'db.php';
```

Comment:
We don't require this file? why is this file removed?

Reply:
This file was added by mistake and is not required in this pull request, hence removing it